lib/inc/drogon/HttpAppFramework.h

```diff
@@ -1449,5 +1496,21 @@ inline HttpAppFramework &app()
 {
     return HttpAppFramework::instance();
 }
-
+#ifdef __cpp_impl_coroutine
+namespace internal
+{
+inline void ForwardAwaiter::await_suspend(
+    std::coroutine_handle<> handle) noexcept
+{
+    app_.forward(
+        req_,
+        [this, handle](const drogon::HttpResponsePtr &resp) {
+            setValue(std::move(resp));
```

Comment:
Don't need `move.` We can't move from a const reference. This just ended up calling the copy constructor.